From d8109a0f95145bdb5f143cf8abab32f83519bb89 Mon Sep 17 00:00:00 2001
From: Balaji Selvanathan <quic_bselvana@quicinc.com>
Date: Mon, 17 Feb 2025 16:59:54 +0530
Subject: [PATCH] PENDING: Change to enable flashing of boot_a and system
 partitions

Made to change to enable flashing of boot_a and system partitions with boot and system images respectively, by storing the fastboot data buffer in different RAM region (with enough space)

Signed-off-by: Balaji Selvanathan <quic_bselvana@quicinc.com>
Upstream-Status: Pending
---
 configs/qcom_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/qcom_defconfig b/configs/qcom_defconfig
index b94e71ccc23..f71a2472c28 100644
--- a/configs/qcom_defconfig
+++ b/configs/qcom_defconfig
@@ -143,7 +143,7 @@ CONFIG_USB_GADGET_PRODUCT_NUM=0xd00d
 CONFIG_CMD_FASTBOOT=y
 CONFIG_FASTBOOT=y
 CONFIG_USB_FUNCTION_FASTBOOT=y
-CONFIG_FASTBOOT_BUF_ADDR=0xaf000000
+CONFIG_FASTBOOT_BUF_ADDR=0xDB300000
 CONFIG_FASTBOOT_BUF_SIZE=0x7000000
 CONFIG_FASTBOOT_USB_DEV=0
 CONFIG_FASTBOOT_FLASH_UFS=y
-- 
2.34.1

